### DISCUSSION TITLE
Welcome to pong Discussions!

### DISCUSSION THREAD

**Discussion by The-Chorus** [The-Chorus/2D-Table-Tennis#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by IbisLiven** [comment]
Also, please read our [Code of Conduct (PLACEHOLDER)](https://github.com/The-Chorus/2D-Table-Tennis) and [Contribution Guide (PLACEHOLDER)](https://github.com/The-Chorus/2D-Table-Tennis).
If you need support, you can look at our [Support Document (PLACEHOLDER)](https://github.com/The-Chorus/2D-Table-Tennis)
